packages/hub/services/realms-config.ts

```diff
@@ -0,0 +1,25 @@
+import { RealmConfig } from '@cardstack/core/src/interfaces';
+import { join } from 'path';
+
+export const BASE_REALM_CONFIG = {
+  url: 'https://cardstack.com/base/',
+  directory: join(__dirname, '..', '..', 'base-cards'),
```

Comment:
it might be jumping the gun a bit, but I wonder if we'll need to use the `require.resolve` approach here too once we merge the webpack build

Reply:
That's a good question. I'll need to defer to @ef4 on that one, because I'm not sure.